Asus/ZenFoneMaxM2/res/values/config.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <fraction name="config_maximumScreenDimRatio">20.000004%</fraction>
+    <fraction name="config_autoBrightnessAdjustmentMaxGamma">300.0%</fraction>
+    <integer-array name="config_autoBrightnessLcdBacklightValues">
+        <item>7</item>
+        <item>10</item>
+        <item>30</item>
+        <item>44</item>
+        <item>56</item>
+        <item>68</item>
+        <item>86</item>
+        <item>98</item>
+        <item>107</item>
+        <item>115</item>
+        <item>125</item>
+        <item>136</item>
+        <item>146</item>
+        <item>188</item>
+        <item>217</item>
+        <item>241</item>
+        <item>251</item>
+        <item>256</item>
+        <item>260</item>
+        <item>264</item>
```

Comment:
sounds wrong. Auto brightnesss works on Q with this?

Reply:
Again m not pro and everything was extracted from stock so sorry
And I will try to be better next time ty